### PR TITLE
[BEAM-1630] Updates common.py due to a Cython bug.

### PR DESCRIPTION
**Pull request by chamikaramj**
Update to common.py is needed due to https://github.com/cython/cython/issues/2059.

Also does a couple of small corrections to common.pxd.
(1) Instance variable of DoFnInvoker should be of type OutputProcessor
not _OutputProcessor.
(2) Method process_outputs() is in OutputProcessor not _OutputProcessor.

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by chamikaramj** [comment]
R: @robertwb

File: sdks/python/apache_beam/runners/common.py [hunk 1]
```diff
@@ -264,7 +264,9 @@ def __init__(self, output_processor, signature):
 
   def invoke_process(self, windowed_value, restriction_tracker=None,
                      output_processor=None):
-    output_processor = output_processor or self.output_processor
+    if not output_processor:
+      output_processor = self.output_processor
+    assert output_processor
```
Comment (robertwb) [inline sdks/python/apache_beam/runners/common.py]: Do we need these asserts on every element?

Comment (chamikaramj) [reply sdks/python/apache_beam/runners/common.py]: Removed asserts.

**Comment by chamikaramj** [comment]
Thanks. Updated commit message and PR description to include both fixes.

**Comment by robertwb** [comment]
LGTM, thanks.

**Comment by chamikaramj** [comment]
Run Python PreCommit

**Pull request merged**